sample/manifest.yml

```diff
@@ -5,10 +5,17 @@ applications:
 #
 - name: logging-sample-app
   instances: 1
+  buildpack: sap_java_buildpack
```

Comment:
Is it actually beneficial for memory consumption to use the sap_java_buildpack? Maybe it would be better to leave this sample app with the community buildpack, so it can easily be deployed to non-SAP CF installations.

Reply:
When I tested this with the normal CF buildpack, the app crashed during deployment when the memory was less than 592M, so yes, the SAP buildpack is *extremely* beneficial.  If anyone wants to deploy it on a landscape _without_ the SAP buildpack, he's free to edit `manifest.yml` accordingly.